Tests/SentryTests/SentryClientTests.swift

```diff
@@ -277,6 +277,68 @@ class SentryClientTest: XCTestCase {
         eventId.assertIsEmpty()
     }
     
+#if os(iOS) || targetEnvironment(macCatalyst)
```

Comment:
`SENTRY_HAS_UIKIT` will also run in tvOS, do we need to run some of these tests for that platform?